templates/prometheus.yaml

```diff
@@ -9,6 +9,16 @@ spec:
   baseImage: registry.redhat.io/openshift3/prometheus
   tag: v3.11
   containers:
+    - name: blackbox-exporter
+      volumeMounts:
+        - name: configmap-{{ .BlackboxExporterConfigmapName }}
+          mountPath: /etc/blackbox-config
+      image: registry.connect.redhat.com/bitnami/prometheus-blackbox-exporter:0.14.0-rhel-7-r33-2
```

Comment:
Is there an upstream version of this we should use by default,
and allow a downstream version to be specified via the CR?

Reply:
This is the only image that shows up when searching for `blackbox-exporter` in the Red Hat registry. Do you think the image should be customizable given that the monitoring operator is very specific to our use case?

Reply:
I'm thinking of the application-monitoring-operator being used in minishift (not openshift or osd) and if we should use upstream images like https://quay.io/repository/prometheus/blackbox-exporter.

However, we're already pulling images from registry.redhat.io and allowing for a pull secret in the install steps in #42, so I think going all in on RH registries is OK our use case. I'm open to this changing in future if there's a need.